app/main/views.py

```diff
@@ -0,0 +1,10 @@
+from flask import Blueprint, render_template
+
+
+bp = Blueprint('main', __name__, url_prefix='/', template_folder='../templates',
```

Comment:
I'm not entirely clear on why you need a blueprint here. Is it just for easy importing since you're using a very modular app structure? Generally I just use `@app.route`

Reply:
Yeah, in this case the blueprint is just to fit with the modular design. In all honesty, it's probably not necessary, but it might help fix clutter if we add User logins or forms in the "main" routes.

Reply:
Fair enough